config/7.1.0/securebanking/ig/scripts/groovy/SaveIntentIdOnAttributesContext.groovy

```diff
@@ -12,6 +12,11 @@ logger.debug(SCRIPT_NAME + "Running...")
 
 def intentId = JsonValue.json(Json.readJson(contexts.oauth2.accessToken.info.claims)).get("id_token").get("openbanking_intent_id").get("value").asString()
 logger.info(SCRIPT_NAME + "Intent Id value: " + intentId)
+
+if (!intentId) {
+    throw new IllegalStateException("openbanking_intent_id claim could not be extracted from the access token");
+}
```

Comment:
This one needs a proper error message.

IllegalStateException (which in turn results in a 500) is reserved for situations where we have detected that the application has been configured incorrectly (or some programming error).

In this case, we have bad user input. We are expecting a token with some data in it but it's not there, so this should response with a 400 and a proper error msg.

Reply:
Updated the error

Reply:
Thanks